### DISCUSSION TITLE
스터디 - 2주차 스터디 목록

### DISCUSSION THREAD

**Discussion by vi2920va** [vi2920va/FE_interview#20] [General]
## 2주차 스터디 목록
-  2주차에 스터디 하실 거 댓글로 남겨주시면 수정하겠습니다.
- 하나의 주제 끝나고 다음 주제하는 식으로 천천히 늘려나가요 화이팅

###  스터디 일정
-  2021.11.16(화)
- 20:00 ~ 20:30 (팀원들 PR 남긴 거 리뷰 시간)
- 20:30 ~ 21:00 (1:1 모의 면접, 3명일 경우 1:다)
- 21:00 ~ 21:30 (모의 면접 후 모여서 마무리로 회고록 작성 후 해산)
 
###  스터디 모음
송현 
- FrontEnd - 크로스브라우징
- FrontEnd - SSR/CSR
- React - 왜 리액트를 쓰는가 ?

영생
- FrontEnd -  로컬스토리지, 세션, 쿠키

동기
- JavaScript - let과 const
- FrontEnd - JWT 토큰

~~인환~~
~~React - LifeCycle~~

**Comment by vi2920va** [comment]
송현-2주차 스터디 주제
- FrontEnd - 크로스브라우징
- FrontEnd - SSR/CSR
- React - Why React ?

**Comment by whdudtod1273** [comment]
영생-2주차 스터디 주제 
- FrontEnd - 로컬스토리지, 세션, 쿠키

**Comment by bamplee** [comment]
인환-2주차 스터디 주제
  - React - Lifecycle

**Comment by ki7348** [comment]
동기 -2주차 스터디 주제
* JavaScript - let과 const
* Web - JWT 토큰

**Reply by vi2920va** [reply]
동기님 JWT와 토큰 - FrontEnd 폴더에 넣어서 작성해주시면 감사하겠습니다!

**Reply by ki7348** [reply]
넵 ㅎㅎㅎ

**Comment by vi2920va** [comment]
오늘 스터디 소감에 대해 적어주세요 ㅎㅎ

**Reply by ki7348** [reply]
- 토큰과 세션 방법이 각각 언제 사용하는지 공부해야 할 것 같습니다.
- var 키워드에 대해 자세히 공부해야 할 것 같습니다.

**Reply by vi2920va** [reply]
공부했던 것도 다시 처음부터 내용을 천천히 깊게 봐야될거 같습니다.

**Reply by whdudtod1273** [reply]
개발할 때 사용하는 것들에 대한 이유나 문제에 대한 해결 방법에 대한 고민을 항상 해야 할 것 같습니다.

**Reply by bamplee** [reply]
질문/답변 시에 좀더 조리있게 말할수 있게 연습하려고 합니다!
